src/vectorclock.erl

```diff
@@ -124,91 +124,33 @@ set_clock_of_dc(DcId, Time, VectorClock) ->
 from_list(List) ->
     dict:from_list(List).
 
-eq(V1, V2) ->
-    dict:fold( fun(Dcid, Time2, Result) ->
-                       case dict:find(Dcid, V1) of
-                           {ok, Time1} ->
-                               case Time1 =:= Time2 of
-                                   true ->
-                                       Result;
-                                   false ->
-                                       false
-                               end;
-                           error ->
-                               false
-                       end
-               end,
-               true, V2).
+-spec for_all_keys(fun((non_neg_integer(), non_neg_integer()) -> boolean()), vectorclock(), vectorclock()) -> boolean().
+for_all_keys(F, V1, V2) ->
+  %% We could but do not care about duplicate DC keys - finding duplicates is not worth the effort
```

Comment:
Can you explain what you mean with "is not worth the effort"?
Shouldn't it hold that each DC only has one entry?